src/raftstore/coprocessor/dispatcher.rs

```diff
@@ -215,7 +215,8 @@ mod test {
     }
 
     fn new_peer_storage(path: &TempDir) -> PeerStorage {
-        let engine = new_engine(path.path().to_str().unwrap(), DEFAULT_CFS).unwrap();
+        let engine = Arc::new(rocksdb::new_engine(path.path().to_str().unwrap(), DEFAULT_CFS)
+            .unwrap());
```

Comment:
`PeerStorage` depends on `rocksdb::DB` directly, we do not have another level of abstraction here.